robusta_krr/core/integrations/prometheus/metrics_service/mimir_metrics_service.py

```diff
@@ -0,0 +1,45 @@
+from typing import Optional
+
+from kubernetes.client import ApiClient
+from prometrix import MetricsNotFound
+
+from robusta_krr.utils.service_discovery import MetricsServiceDiscovery
+
+from .prometheus_metrics_service import PrometheusMetricsService
+
+class MimirMetricsDiscovery(MetricsServiceDiscovery):
+    def find_metrics_url(self, *, api_client: Optional[ApiClient] = None) -> Optional[str]:
+        """
+        Finds the Mimir Metrics URL using selectors.
+        Args:
+            api_client (Optional[ApiClient]): A Kubernetes API client. Defaults to None.
+        Returns:
+            Optional[str]: The discovered Mimir Metrics URL, or None if not found.
+        """
+        return super().find_url(
+            selectors=[
+                "app.kubernetes.io/name=mimir,app.kubernetes.io/component=query-frontend",
```

Comment:
That selector works for my installation and I don't know any others.  List can be extended, of course.